extensions/cache/src/main/java/org/radargun/stages/cache/test/CacheInvocations.java

```diff
@@ -599,4 +606,98 @@ public Operation txOperation() {
          return async ? ASYNC_TX : NATIVE_TX;
       }
    }
+
+   public static final class GetViaStream<K, V> implements Invocation<V> {
+      public static final Operation GET_NULL = StreamingOperations.GET.derive("Null");
+      public static final Operation TX = StreamingOperations.GET.derive("TX");
+      private final StreamingOperations.StreamingCache<K> cache;
+      private final K key;
+      private V value;
+
+      public GetViaStream(StreamingOperations.StreamingCache<K> cache, K key) {
+         this.cache = cache;
+         this.key = key;
+      }
+
+      @Override
+      public V invoke() {
+         InputStream in;
+         Integer result = 0;
+         if ((in = cache.getViaStream(key)) == null)
+            return null;
+
+         try {
+            int read = 0;
+            byte[] arr = new byte[100];
+            while ((read = in.read(arr)) != -1) {
+               result = result + read;
+            }
+            in.close();
+         } catch (IOException e) {
+            e.printStackTrace();
```

Comment:
Yes, that should be rather recorded as error. And please check for other uses of `e.printStackTrace()`.